js/utils/fakeServer.js

```diff
@@ -18,8 +19,11 @@ var server = {
     // populates the localStorage
     if (localStorage.users === undefined) {
       // Set default user
+      const AzureDiamond = "AzureDiamond";
+      const AzureDiamondSalt = genSalt(AzureDiamond);
+      const AzureDiamondPass = bcrypt.hashSync("hunter2", AzureDiamondSalt);
       users = {
-        "AzureDiamond": bcrypt.hashSync("hunter2", salt)
+        [AzureDiamond]: bcrypt.hashSync(AzureDiamondPass, salt)
```

Comment:
This is a new feature in ES6 — dynamic properties. Can be here and not necessarily https://github.com/lukehoban/es6features#enhanced-object-literals